test/e2e/csv_e2e_test.go

```diff
@@ -4289,39 +4287,12 @@ func buildAPIServiceCleanupFunc(c operatorclient.ClientInterface, apiServiceName
 	}
 }
 
-func createCRD(c operatorclient.ClientInterface, crd apiextensions.CustomResourceDefinition) (cleanupFunc, error) {
-	out := &apiextensionsv1.CustomResourceDefinition{}
-	scheme := runtime.NewScheme()
-	if err := apiextensions.AddToScheme(scheme); err != nil {
-		return nil, err
-	}
-	if err := apiextensionsv1.AddToScheme(scheme); err != nil {
-		return nil, err
-	}
-	if err := scheme.Convert(&crd, out, nil); err != nil {
-		return nil, err
-	}
-	_, err := c.ApiextensionsInterface().ApiextensionsV1().CustomResourceDefinitions().Create(context.TODO(), out, metav1.CreateOptions{})
-	if err != nil {
-		return nil, err
-	}
-
-	return buildCRDCleanupFunc(c, crd.GetName()), nil
+func createCRD(c operatorclient.ClientInterface, crd apiextensionsv1.CustomResourceDefinition) (cleanupFunc, error) {
+	return createV1CRD(c, crd)
 }
```

Comment:
This could just be removed now and replaced with `createV1CRD()` everywhere it's used.

Reply:
made that change in #3067